codegen/src/main/java/software/amazon/awssdk/codegen/poet/PoetExtension.java

```diff
@@ -25,11 +25,18 @@
 /**
  * Extension and convenience methods to Poet that use the intermediate model.
  */
-public class PoetExtensions {
+public class PoetExtension {
+
+    public static final String ASYNC_STREAMING_INPUT_PARAM = "requestBody";
+    public static final String ASYNC_STREAMING_OUTPUT_PARAM = "asyncResponseTransformer";
+    public static final String SYNC_STREAMING_INPUT_PARAM = "requestBody";
+    public static final String SYNC_STREAMING_OUTPUT_PARAM = "responseTransformer";
+    public static final String EVENT_PUBLISHER_PARAM_NAME = "requestStream";
+    public static final String EVENT_RESPONSE_HANDLER_PARAM_NAME = "asyncResponseHandler";
```

Comment:
Why are these poet-related?

Reply:
Nope, I can move them to a different class.